app/views/exchanges/employer_applications/index.js.erb

```diff
@@ -255,7 +278,7 @@ function addTermReasons(reason, id) {
   $.ajax({
     type: 'get',
     datatype: 'js',
-    url: '/exchanges/employer_applications/get_term_reasons',
+    url: '/exchanges/employer_applications/term_reasons',
```

Comment:
is this required?

Reply:
It's because of rubocop issue I have updated the method name.